scripts/build_utils.sh

```diff
@@ -1680,11 +1680,7 @@ maybe_reset_ci_container() {
 }
 
 # NOTE: These functions will only work on a Pull Request job!
-pr_only_for() {
-  # $1 is passed by reference to avoid having to call with ${array[@]} and
-  # receive by creating another local array ("$@")
-  local -n local_patterns=$1
-  local files
+pr_filenames_changed() {
```

Comment:
ex https://jenkins.ceph.com/view/all/job/preserve-ceph-pull-requests/2/consoleFull for https://github.com/ceph/ceph/pull/63078

i'm seeing this extra path listed 3 times
```
changed files:
~/build/workspace/preserve-ceph-pull-requests ~/build/workspace/preserve-ceph-pull-requests
Dockerfile.build
~/build/workspace/preserve-ceph-pull-requests
```
which would cause the docs-only/container-only checks to fail regardless

Reply:
oops, that extra stuff is in stdout from `pushd`/`popd`